index.html

```diff
@@ -1,140 +0,0 @@
-<!DOCTYPE html>
-<html lang="en">
-    <head>
-        <meta charset="UTF-8">
-        <meta http-equiv="X-UA-Compatible" content="IE=edge">
-        <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <title>DrivenEats</title>
-        <link rel="stylesheet" href="reset.css">
-        <link rel="stylesheet" href="style.css">
-        <link rel="preconnect" href="https://fonts.googleapis.com">
-        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-        <link href="https://fonts.googleapis.com/css2?family=Righteous&family=Roboto:wght@300;400;700&display=swap" rel="stylesheet">
-        <link rel="shortcut icon" href="images/header.png"/>
-        
-    </head>
-    <body>
-        <div class="tela-de-confirmacao">
-            <div class="tela-verde" data-identifier="confirmation-dialog">
-                <h1>Confirme seu pedido</h1>
```

Comment:
`tela-verde`? Que tela é essa? É que nem a tela azul do Windows? Onde vive? Do que se alimenta?

lembrar de colocar nomes mais semanticos nas classes, funções e variaveis, assim quando algum outro programador for olha no seu codigo ele vai conseguir identificar de cara do que se trata.